Games/types/Mafia/roles/cards/ConvertSave.js

```diff
@@ -0,0 +1,31 @@
+const Card = require("../../Card");
+const { PRIORITY_NIGHT_SAVER, PRIORITY_CONVERT_DEFAULT } = require("../../const/Priority");
+
+module.exports = class ConvertSave extends Card {
+  constructor(role) {
+    super(role);
+
+    this.meetings = {
+      "Save": {
+        states: ["Night"],
+        flags: ["voting"],
+        action: {
+          labels: ["save"],
+          priority: PRIORITY_NIGHT_SAVER, PRIORITY_CONVERT_DEFAULT,

```

Comment:
this code does not run, please test and insert basic screenshots before requesting for review